akka-stream/src/main/scala/akka/stream/io/OutputStreamSource.scala

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.io
+
+import java.io.OutputStream
+
+import akka.stream._
+import akka.stream.impl.io.OutputStreamSource
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+/**
+ * Source which allows to use [[OutputStream]] to interact with reactive stream.
+ */
+object OutputStreamSource {
+  import scala.language.postfixOps
+
+  final val DefaultChunkSize = 8192
```

Comment:
will remove it as I'm not doing chunks anymore. assuming that there is some contract about ByteString size.